tests/app/main/test_frameworks.py

```diff
@@ -3776,6 +3776,38 @@ def test_should_be_a_503_if_email_fails(self, send_email, data_api_client):
 @mock.patch('app.main.views.frameworks.data_api_client', autospec=True)
 @mock.patch('app.main.views.frameworks.count_unanswered_questions')
 class TestG7ServicesList(BaseApplicationTest):
+    def _assert_incomplete_application_banner_not_visible(self, html):
+        assert "Your application is not complete" not in html
```

Comment:
Would also be nice to have the exact opposite assertion, ie `assert "Your application is not complete" in html`, in the tests for when banner is visible.

Reply:
Done.